javascript/src/main/pages/Home/Home.js

```diff
@@ -37,7 +37,7 @@ const Home = () => {
             <div className="text-left">
                 <h5>Welcome to the UCSB Courses Search App!</h5>
                 <BasicCourseSearchForm setCourseJSON={setCourseJSON} fetchJSON={fetchBasicCourseJSON} />
-                <Button><CSVLink style={{color: "white"}} headers={courseHeaders} data={courseJSON.classes} filename = {"CourseTable.csv"}>Download CSV</CSVLink></Button>
+                <Button><CSVLink style={{color: "white", marginTop: "1rem"}} headers={courseHeaders} data={courseJSON.classes} filename = {"CourseTable.csv"}>Download CSV</CSVLink></Button>
```

Comment:
I'm not noticing any difference between the "Download CSV" button in your change (on the left) and the button in the current production app / main branch (on the right). See the screenshot below.

![image](https://user-images.githubusercontent.com/34492835/108981696-d6956980-7641-11eb-8403-ed509df5ac2d.png)

It looks like you're adding the margin to the wrong component - instead of adding it to the link, we should be adding it to the button. Can we make this fix?

Also, instead of applying the margin only to the top, can we apply it to the bottom as well, so the button doesn't touch the top of the results table? Instead of using `margin-top: 1rem`, we can use `margin: 1rem 0`, since margin with two arguments sets the first argument to top/bottom and the second argument to left/right. Ideally, it should look something like this:

![image](https://user-images.githubusercontent.com/34492835/108982051-39870080-7642-11eb-94a5-c442bffd7a36.png)

Reply:
Andrew, I fixed the problem with the latest commit.  If you could take a look to make sure the change is proper. The home page on this branch deploy now looks like this:

<img width="1195" alt="Screen Shot 2021-02-24 at 3 15 17 PM" src="https://user-images.githubusercontent.com/59560618/109079311-9e2e7380-76b3-11eb-8a72-ff6048eda7be.png">